packages/vsphere/data_stream/network/fields/fields.yml

```
@@ -0,0 +1,72 @@
+- name: vsphere.network
+  type: group
+  description: >
+    Network-related information.
+  fields:
+    - name: accessible
+      type: boolean
+      description: >
+        Indicates whether at least one host is configured to provide this network.
+    - name: config.status
+      type: keyword
+      description: >
+        Indicates whether the system has detected a configuration issue.
+    - name: host
+      type: group
+      fields:
+        - name: names
+          type: keyword
+          description: >
+            Names of the hosts connected to this network.
+        - name: count
+          type: long
+          metric_type: gauge
+          description: >
+            Number of hosts connected to this network.
+    - name: name
+      type: keyword
+      description: >
+        Name of the network.
+    - name: status
+      type: keyword
+      description: >
+        General health of the network.
+    - name: type
+      type: keyword
+      description: >
+        Type of the network (e.g., Network(Standard), DistributedVirtualport).
+    - name: vm
+      type: group
+      fields:
+        - name: names
+          type: keyword
+          description: >
+            Names of the virtual machines connected to this network.
```

```suggestion
            List of all the virtual machines connected to this network.
```

minor nit. Please do it for other similar fields as well.